src/components/Bookmark.js

```diff
@@ -1,17 +1,27 @@
 import React from 'react'
 import styled from 'styled-components'
+import Tooltip from '@material-ui/core/Tooltip'
 
-export default function Bookmark({ onBookmarkClick, id, bookmarkStatus }) {
+export default function Bookmark({
+  onBookmarkClick,
+  id,
+  bookmarkStatus,
+  pathname,
+}) {
   function handleBookmarkClick(event) {
     event.stopPropagation()
     onBookmarkClick(id)
   }
 
   return (
-    <BookmarkStyled
-      onClick={event => handleBookmarkClick(event)}
-      active={bookmarkStatus}
-    ></BookmarkStyled>
+    <>
```

Comment:
You don't need the '<>' because your code is already wrapped into Tooltip.

Reply:
True, thank  you. Forgot to delete it.